### DISCUSSION TITLE
So, did anyone manage to train and get good voices or not?

### DISCUSSION THREAD

**Discussion by Subarasheese** [lifeiteng/valle#38] [General]
I am in the middle of training the model now, but I've seen comments from people that trained but got bad results.

Can anyone confirm whether those were isolated cases, or if there is indeed a problem with training?

I haven't seen anyone talking about success and/or good results... In case someone successfully trained, mind to share a Vocaroo sample of the result?

**Comment by MisakaMikoto96** [comment]
get good performance on Mandarin yet.
